charts/dex-k8s-authenticator/templates/deployment.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         release: {{ .Release.Name }}
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    {{- with .Values.annotations }}
```

Comment:
Could we call this `podAnnotations` - since these are only applied to the pod-spec and not top-level deployment.

In addition could we provide defaults in `values.yaml` to make it clear it's an option, someting like:


```
podAnnotations: { }

nodeSelector: {}

tolerations: []

affinity: {}
```

Reply:
Hi @nabadger, sorry for my late reply. Yes we could do that exactly as you described. So `podAnnotations` is fine as well as the definition of `podAnnotations: { } in the value file.